docs/components/encoder/incremental-micro-rdk.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Configure an Incremental Encoder (Micro-RDK)"
+title: "Incremental Encoder (viam-micro-server)"
```

Comment:
Why is the "Configure a" going? If the title is too long lmk and we may need to oberweite that setting